portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -50,11 +53,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
     FetchOptions fetchOptions = FetchOptions.Builder.withLimit(maxComments);
 
-    ArrayList<String> comments = new ArrayList<>();
+    ArrayList<HashMap> comments = new ArrayList<>();
```

Comment:
Hashmap works pretty well for now but its restrict it two the specific key and value pair types. It would be extensible if you create a new Comment class with properties being in this case text and email but could extend to any other properties with different types.